### PR TITLE
New port: Demon Keep

### PR DESCRIPTION
**Pull request by mxmgorin**
# New Port for Demon Keep

## Game Information
- **Title**: Demon Keep
- **URL**: https://youdoyoubuddy.itch.io/demon-keep

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] JELOS
- [X] ROCKNIX

### Resolution Tests
Test all major resolutions:
- [X] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Review by Cebion — APPROVED** [review]

File: .idea/.idea.PortMaster-New.dir/.idea/indexLayout.xml
Comment (kloptops) [inline .idea/.idea.PortMaster-New.dir/.idea/indexLayout.xml:1]: This file again. 😁

**Review by kloptops — CHANGES_REQUESTED** [review]
Just need to delete that `.idea` directory. 😄

File: .idea/.idea.PortMaster-New.dir/.idea/indexLayout.xml
Comment (mxmgorin) [reply .idea/.idea.PortMaster-New.dir/.idea/indexLayout.xml:1]: deleted

**Review by kloptops — APPROVED** [review]
👍

**Pull request merged**